src/plugins/intel_cpu/tests/functional/subgraph_tests/src/subgraph_select_pd.cpp

```diff
@@ -0,0 +1,98 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/opsets/opset8.hpp"
+#include "ov_models/builders.hpp"
+#include "shared_test_classes/base/ov_subgraph.hpp"
+#include "test_utils/cpu_test_utils.hpp"
+
+namespace ov {
+namespace test {
+
+class SubgraphSelectPD : virtual public SubgraphBaseStaticTest {
+protected:
```

Comment:
Could you please add ascii subgraph picture with expected behavior.

Reply:
Not necessary for a release branch, only for master

Reply:
I will create same pr to master. but customer request me to create pr to 2023.3 at firstly. @EgorDuplensky